src/imitation/rewards/reward_nets.py

```diff
@@ -367,6 +555,14 @@ def __init__(
             discount_factor: discount factor for the potential shaping.
             kwargs: passed straight through to `BasicRewardNet` and `BasicPotentialMLP`.
         """
+        # FIXME(yawen): why could the reward net and potential net use the same kwargs
```

Comment:
We do specify the hidden sizes of the reward & potential net separately. Most of the other parameters don't make sense to change from default at all anyway, but the ones I do see as being reasonable to configure (e.g. `activation`, `normalize_input_layer`) you probably do usually want shared between the networks.

So I think this is reasonable, although if you want to make it so you can specify different kwargs for each I'd be fine with that too so long as there's a way to specify kwargs shared for both.

Reply:
Seems reasonable. I was mostly worried about `kwargs` fed to `.build_mlp_kwargs` might contain unwanted arguments like `normalize_output_layer`. But now the problem doesn't exist because we moved the initialization of normalize_output_layer to `reward.make_reward_net`.